cmd/thanos/receive.go

```diff
@@ -859,6 +868,9 @@ func (rc *receiveConfig) registerFlag(cmd extkingpin.FlagClause) {
 
 	cmd.Flag("receive.replica-header", "HTTP header specifying the replica number of a write request.").Default(receive.DefaultReplicaHeader).StringVar(&rc.replicaHeader)
 
+	compressionOptions := strings.Join([]string{compressionNone, snappy.Name}, ", ")
+	cmd.Flag("receive.grpc-compression", "Compression algorithm to use for gRPC requests to other receivers. Must be one of: "+compressionOptions).Default(compressionNone).EnumVar(&rc.compression, compressionNone, snappy.Name)
```

Comment:
Maybe we should enable it by default here? 🙂

Reply:
Yeah, I agree. Enabling it by default makes more sense.

Reply:
I was going to suggest that for the v1 release, but even now it would make sense given the favourable tradeoff. Updated to default 👍